src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <h1></h1>
```

Comment:
If you have removed the text don't leave an empty tags. Or just remove the tag or keep the text.